Android/src/org/droidplanner/android/dialogs/UVCDialog.java

```diff
@@ -0,0 +1,209 @@
+package org.droidplanner.android.dialogs;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.hardware.usb.UsbDevice;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.Button;
+import android.widget.CheckedTextView;
+import android.widget.Spinner;
+
+import com.serenegiant.usb.DeviceFilter;
+import com.serenegiant.usb.USBMonitor;
+
+import org.droidplanner.android.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+public class UVCDialog extends DialogFragment {
```

Comment:
@guiseco can you make use of the `DialogFragment` from the support library instead.